internal/agent/reset.go

```diff
@@ -71,49 +139,51 @@ func Reset(reboot, unattended bool, dir ...string) error {
 
 	bus.Manager.Publish(sdk.EventBeforeReset, sdk.EventPayload{}) //nolint:errcheck
 
-	c, err := config.Scan(collector.Directories(dir...))
-	if err != nil {
-		return err
+	// Prepare a config from the cli flags
+	r := map[string]map[string]interface{}{
```

Comment:
I think a struct would make more sense, to me it feels a bit strange to be doing too much stuff like this: `r["reset"]["reset-oem"]` and maybe I'm mistaken but then it would be easier to deduce because this would be automatic: `c.Reset.ResetOem = resetOem`

Reply:
This is a good point, but this is then marshalled into a string to be passed to the collector, so I'm not sure how would it fit into that as it needs to be a reader for the collector to read it as another source.

Reply:
well, that was simpler that I expected LMAO